ADVANCE_SPEED: 0.1
TURNING_SPEED: 0.35